src/network/networkd-address.c

```diff
@@ -479,7 +479,7 @@ int address_remove(
                 return log_link_error_errno(link, r, "Could not append IFA_LOCAL attribute: %m");
 
         r = netlink_call_async(link->manager->rtnl, NULL, req,
-                               callback ?: address_remove_handler,
+                               callback ? callback: address_remove_handler,
```

Comment:
The change is completely equivalent, and we use the shorter version. So, please drop this.